tests/integration/test_fim/test_synchronization/test_registry_responses_win32.py

```diff
@@ -187,20 +141,22 @@ def test_registry_sync_after_restart(key_name, value_name, tags_to_apply,
         ValueError: If a path or value are not in the sync event.
     """
     check_apply_test(tags_to_apply, get_configuration['tags'])
-
     key_path = os.path.join(subkey, key_name)
     value_path = os.path.join(key, key_path, value_name)
 
     # wait until the sync is done.
-    get_sync_msgs(sync_interval)
+    wazuh_log_monitor.start(timeout=sync_interval + 15, callback=fim.callback_detect_registry_integrity_clear_event,
+                            error_message='Did not receive expected "integrity clear" event')
```

Comment:
We should probably move this validation to the fixture in charge of removing the created registry entry.